lib/cog_api/fake/client.ex

```diff
@@ -84,6 +86,18 @@ defmodule CogApi.Fake.Client do
     Roles.revoke(endpoint, role, group)
   end
 
+  def rule_index(command, %Endpoint{}=endpoint) do
```

Comment:
Can we either: a) not switch the order of these yet, or b) switch the order of everything (fine if its in another PR sometime today or Wednesday, but I'd rather not leave it too long)